jvm/hermes/src/main/jni/JHermesRuntime.h

```diff
@@ -14,9 +14,8 @@ class JHermesConfig : public HybridClass<JHermesConfig> {
     static constexpr auto kJavaDescriptor = "Lcom/intuit/playerui/hermes/bridge/runtime/HermesRuntime$Config;";
     static void registerNatives();
 
-    static local_ref<jhybridobject> create(alias_ref<jclass>, bool intl = true, bool microtaskQueue = false) {
+    static local_ref<jhybridobject> create(alias_ref<jclass>, bool intl = false, bool microtaskQueue = false) {
         auto config = hermes::vm::RuntimeConfig::Builder()
-            .withIntl(intl)
```

Comment:
Either keep passing the param as config, or remove the param altogether - don't need more places where seemingly useful config params aren't actually used